dev/doc/INSTALL.dune.md

```diff
@@ -89,7 +80,7 @@ Note that for a fast developer build of ML files, the `check` target
 is faster, as it doesn't link the binaries and uses the non-optimizing
 compiler.
 
-If you built the full standard library with the `world` target,
+If you built the full standard library with the default target,
```

Comment:
Is that correct?  I always use `world`.  What is the default target--do you mean ``` `default` ```?

Reply:
The default target, invoked via `dune build` builds the standard library.